### PR TITLE
iz list & localize notebooks

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

**Comment by zarsky-broad** [comment]
Also #58

**Review by MattPutnam — APPROVED** [review]

File: src/pages/workspaces/workspace/Container.js [hunk 1]
```diff
@@ -128,7 +134,11 @@ const WorkspaceContainer = hh(class WorkspaceContainer extends Component {
           ]
         )
       ]),
-      tabComponents[activeTab]({ namespace, name, key: forceUpdateKey })
+      Utils.cond(
+        [workspaceFailure, ''],
```
Comment (dmohs) [inline src/pages/workspaces/workspace/Container.js]: No error message? I still think we need a more general solution for this—I see fatigue here from constantly having to handle this case.

File: src/pages/workspaces/workspace/Notebooks.js [hunk 2]
```diff
@@ -51,93 +53,168 @@ export default hh(class WorkspaceNotebooks extends Component {
...
 
+  getNotebooks() {
+    this.setState({ notebooks: undefined, notebookAccess: {} })
+    const { workspace } = this.props
+
+    Buckets.listNotebooks(workspace.bucketName,
+      notebooks => {
+        const cluster = _.first(this.state.clusters).clusterName
+
+        this.setState({ notebooks: _.reverse(_.sortBy(notebooks, 'updated')) })
```
Comment (dmohs) [inline src/pages/workspaces/workspace/Notebooks.js:64]: Good choice.

File: src/pages/workspaces/workspace/Notebooks.js [hunk 3]
```diff
@@ -51,93 +53,168 @@ export default hh(class WorkspaceNotebooks extends Component {
...
+                  }, name.slice(10, -6)), // ignores 'notebooks/' and the .ipynb suffix
+                  icon('jupyterIcon',
+                    { style: { height: 125, width: 'auto', color: Style.colors.background } }),
+                  div({ style: { display: 'flex', alignItems: 'flex-end' } }, [
+                    div({ style: { fontSize: '0.8rem', flexGrow: 1 } }, [
+                      'Last changed:',
+                      div({}, Utils.makePrettyDate(updated))
+                    ]),
+                    Utils.cond(
+                      [notebookAccess[name] === false, () => icon('times', { title: 'Error' })],
```
Comment (dmohs) [inline src/pages/workspaces/workspace/Notebooks.js]: I liked this when I saw it, but even better might be to link it to the localization.log in the cluster.

**Review by dmohs — APPROVED** [review]
A filed bug is nearly as good as addressing an issue ;)

File: src/pages/workspaces/workspace/Notebooks.js (hunk 3 above)
Comment (zarsky-broad) [reply src/pages/workspaces/workspace/Notebooks.js]: Once we have the synchronous version of this call, this will be moot.

**Review by dmohs — APPROVED** [review]
👍

**Pull request merged**